src/flyte/models.py

```diff
@@ -410,13 +409,7 @@ class SerializationContext:
     code_bundle: Optional[CodeBundle] = None
     input_path: str = "{{.input}}"
     output_path: str = "{{.outputPrefix}}"
-    _entrypoint_path: str = field(default="_bin/runtime.py", init=False)
+    interpreter_path: str = "/opt/venv/bin/python"
+    entrypoint_path: str = "/opt/venv/bin/runtime.py"
     image_cache: ImageCache | None = None
     root_dir: Optional[pathlib.Path] = None
-
-    def get_entrypoint_path(self, interpreter_path: str) -> str:
```

Comment:
can we still keep this. I do not want both interpreter_path and entrypoint path to be set. Can we also compute entrypoint_path from interpreter path? 
```python
@property
def entrypoint_path():
      return self.interpreter_path.parent / "runtime.py"
```